model/riscv_insts_zkn_rv32.sail

```diff
@@ -0,0 +1,170 @@
+
```

Comment:
Apologies, dumb question time: How do I get the low `XLEN` bits of a 64-bit value? In my naivety I tried
`result[xlen..0]` and `result[sizeof(xlen)..0]`, but no. I'm unaware of another example of this in the architecture.

Reply:
Meta comment - this feels like it's going to make the Sail code much more counter-intuitive to read.

Reply:
Yeah, but if UXL are ever to be supported then I don't think there's a way to avoid that.

To answer your question, it's `result[(sizeof(xlen) - 1) .. 0]`; see the code to read mcycle/minstret for examples of needing this elsewhere in the model, though it's fairly rare.